src/KubeOps/Operator/Webhooks/KubernetesJsonDiffer.cs

```diff
@@ -1,27 +1,27 @@
-﻿using JsonDiffPatch;
+﻿using System.Text.Json.JsonDiffPatch;
+using System.Text.Json.JsonDiffPatch.Diffs.Formatters;
+using System.Text.Json.Nodes;
 using k8s;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
 namespace KubeOps.Operator.Webhooks;
 
 internal static class KubernetesJsonDiffer
 {
-    private static readonly JsonDiffer JsonDiffer = new();
+    private static readonly JsonPatchDeltaFormatter Formatter = new();
 
-    public static PatchDocument DiffObjects(object? from, object? to)
+    public static JsonNode? DiffObjects(object? from, object? to)
```

Comment:
By default, `JsonNode.ToString()` returns the indented form, which is the same as the old `PatchDocument`.